love/src/Config.js

```diff
@@ -336,9 +336,9 @@ export const m1VentsLimitSwitchesStateMap = {
 
 // ATHexapod
 export const hexapodInPositionStateMap = {
-  1: 'NOT READY',
-  2: 'IN POSITION',
-  0: 'UNKNOWN',
+  1: 'IN POSITION',
+  2: 'UNKNOWN',
+  0: 'NOT READY',
```

Comment:
Here the mapping should be `1: 'IN POSITION'`, `2: 'NOT READY'` and `3: 'UNKNOWN'` (1 are 2 are the expected values, 0 is when we don't receive data)